config.xml

```diff
@@ -5,8 +5,48 @@
   <numExecutors>2</numExecutors>
```

Comment:
can we switch and default to using the insecure config? Once the instances self update and pull this file down, they'll have to switch to the other file every time.